### PR TITLE
Make is possible for staff to set all workflow states on submissions,…

### LINKED ISSUE #853: Unable to "dismiss" or "invite to propsal" any concept note [High/Bug]
-=Sensitive=-
No

-=Reporter=-

-=Ideal deadline=-

-=Who's affected=-
OTF team, Applicants

-=Description=-
After testing on multiple concept notes under 2019-03 for IFFund, I am unable to progress any of the concept notes to a state that is not "Request more information". I need to dismiss and invite to proposal several concept notes under the 2019-01 round, and am unable to do so.  

My understanding was that this should have been resolved based on this ticket: https://github.com/OpenTechFund/opentech.fund/issues/725

-=Objective=-

-=Additiional info=-

-=Timestamp=-
2019-01-16T16:03:35.703Z

### LINKED ISSUE #855: All team members (not just round lead) should be able to send "request for information", "dismiss", "invite to proposal" etc [Medium/Improvement]
-=Sensitive=-
No

-=Reporter=-

-=Ideal deadline=-

-=Who's affected=-
OTF team

-=Description=-
Currently, only the round lead has the capability to send "request for information", "dismiss", "invite to proposal" etc for concept notes and proposals. Any team member should be able to send these statuses. 

-=Objective=-

-=Additiional info=-

-=Timestamp=-
2019-01-16T17:44:52.204Z

### PR DESCRIPTION
**Pull request by frjo**
… not only the lead as it's now.

Fixes #853
Fixes #855

### CODE REVIEW

File: opentech/apply/funds/workflow.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def __init__(self, name, display, stage, permissions, step, transitions=dict()):
         # For building transition methods on the parent
         self.transitions = {}
 
-        default_permissions = {UserPermissions.STAFF, UserPermissions.ADMIN, UserPermissions.LEAD}
+        default_permissions = {UserPermissions.STAFF, UserPermissions.LEAD, UserPermissions.ADMIN}
```
Comment (frjo) [inline opentech/apply/funds/workflow.py]: Just my itch to have all lists in the same order.

**Comment by danblah** [comment]
Bumping this.

File: opentech/apply/determinations/tests/test_views.py [hunk 2]
```diff
@@ -79,12 +74,6 @@ def test_cant_edit_submitted_more_info(self):
         response = self.get_page(submission, 'form')
         self.assertNotContains(response, 'Update ')
 
-    def test_cannot_edit_draft_determination_if_not_lead(self):
```
Comment (todd-dembrey) [inline opentech/apply/determinations/tests/test_views.py]: Can we flip this test `test_can_edit_draft_determination_if_not_lead`

File: opentech/apply/determinations/tests/test_views.py [hunk 3]
```diff
@@ -49,11 +49,6 @@ def test_can_access_form_if_lead(self):
         self.assertContains(response, submission.title)
         self.assertContains(response, submission.get_absolute_url())
 
-    def test_cant_access_wrong_status(self):
-        submission = ApplicationSubmissionFactory(status='more_info')
```
Comment (todd-dembrey) [inline opentech/apply/determinations/tests/test_views.py]: perhaps keep this with an already determined state? e.g. `concept_rejected`

**Comment by frjo** [comment]
@danblah This should now be possible to deploy. Just to make sure I got it right…

This PR will give staff the same permissions regarding setting state as lead used to have.

All staff will be able to set any state on any submission! That is what you want, correct?

**Pull request merged**